configure.ac

```diff
@@ -1641,10 +1647,14 @@ AC_MSG_RESULT($build_bitcoin_chainstate)
 
 AC_MSG_CHECKING([whether to build libraries])
 AM_CONDITIONAL([BUILD_BITCOIN_LIBS], [test $build_bitcoin_libs = "yes"])
+
 if test "$build_bitcoin_libs" = "yes"; then
   AC_DEFINE([HAVE_CONSENSUS_LIB], [1], [Define this symbol if the consensus lib has been built])
   AC_CONFIG_FILES([libbitcoinconsensus.pc:libbitcoinconsensus.pc.in])
 fi
+
+AM_CONDITIONAL([BUILD_BITCOIN_KERNEL_LIB], [test "$build_experimental_kernel_lib" != "no" && ( test "$build_experimental_kernel_lib" = "yes" || test "$build_bitcoin_chainstate" = "yes" )])
```

Comment:
> I don't remember if `bitcoin-chainstate` is intended to be a long-term binary though you could add a rule in the top makefile to build it from the root bitcoind directory. I was confused to fail on it at first.

It could be convenient to add the rule, but I think think this is not intended to stick around long term. Instead it will be moved to an examples folder and given a cmake file (:partying_face:) and I think just detached from the main build: https://github.com/dongcarl/bitcoin/tree/2022-01-v8-on-new-kirby/src/kernel/example/bitcoin-chainstate